bin/files.js

```diff
@@ -35,35 +35,90 @@ exports.readManifest = (key) => {
   return manifest[key];
 };
 
-exports.findReplace = (findString, replaceString) => {
+exports.findReplace = async(findString, replaceString) => {
   const regex = new RegExp(findString, 'g');
   const options = {
     files: `${exports.rootDir}/**/*`,
     from: regex,
     to: replaceString,
     ignore: [
-      `${exports.rootDir}/node_modules/**/*`,
-      `${exports.rootDir}/.git/**/*`,
-      `${exports.rootDir}/.github/**/*`,
-      `${exports.rootDir}/vendor/**/*`,
-      `${exports.rootDir}/_rename.sh`,
-      `${exports.rootDir}/bin/rename.js`,
-      `${exports.rootDir}/bin/setup-wp.js`,
-      `${exports.rootDir}/theme-manifest.js`,
+      path.join(`${exports.rootDir}/node_modules/**/*`),
+      path.join(`${exports.rootDir}/.git/**/*`),
+      path.join(`${exports.rootDir}/.github/**/*`),
+      path.join(`${exports.rootDir}/vendor/**/*`),
+      path.join(`${exports.rootDir}/wp-admin/**/*`),
+      path.join(`${exports.rootDir}/wp-includes/**/*`),
+      path.join(`${exports.rootDir}/_rename.sh`),
+      path.join(`${exports.rootDir}/bin/rename.js`),
+      path.join(`${exports.rootDir}/bin/setup-wp.js`),
+      path.join(`${exports.rootDir}/bin/output.js`),
+      path.join(`${exports.rootDir}/bin/files.js`),
+      path.join(`${exports.rootDir}/theme-manifest.js`),
     ],
   };
 
-  try {
-    const changes = replace.sync(options);
+  return new Promise((resolve) => {
```

Comment:
Above you used `async` so how come you used a `Promise` here? Couldn't this be also made using `async/await`?

Reply:
It was a c/p (first thing I converted to `async`) and when it worked, I didn't bother with it anymore later. Will rework to async. 👍